resizeobserver/demo.js

```diff
@@ -0,0 +1,13 @@
+/* global ResizeObserver */
+
+const ro = new ResizeObserver(entries => {
```

Comment:
That's because border is being changed in observer callback, which triggers another resize event immediately.